devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -58,17 +64,32 @@ metadata {
 }
 
 def updated() {
-    stopTimer()
-    startTimer()
+	log.debug "updated()"
+    if (device.getDataValue("manufacturer") == "SmartThings"){
+		stopTimer()
```

Comment:
Please, could You fix this indentation ?

Reply:
OK! I will fix this indentation.